frontend/nextjs/components/ResearchBlocks/ChatInterface.tsx

```
@@ -0,0 +1,221 @@
+                  } relative overflow-hidden group transition-all duration-300 hover:shadow-lg`}
+                >
+                  {/* Add subtle animated gradient effect */}
+                  <div className={`absolute inset-0 opacity-10 group-hover:opacity-20 transition-opacity duration-300 ${
+                    msg.role === 'user' 
+                      ? 'bg-gradient-to-br from-teal-300/40 via-transparent to-cyan-300/30' 
+                      : 'bg-gradient-to-br from-indigo-400/30 via-transparent to-purple-400/20'
+                  } animate-pulse pointer-events-none`}></div>
+                  
+                  <div 
+                    className="markdown-content text-sm sm:text-base relative z-10"
+                    dangerouslySetInnerHTML={{ __html: msg.html }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)